### ISSUE TITLE
Improper syntax (likely?)

### ISSUE THREAD

**Issue by DanHolli123** [sephcasiah/xirakado#2]
It'd be nice to  have it check for folders so those aren't seen to make it cleaner
trying to process `C:\Program`
and too short of commands

`The system cannot find the file specified.
A subdirectory or file C:\Users\Taylor\AppData\Roaming\xirakado already exists.
A subdirectory or file C:\Users\Taylor\AppData\Roaming\xirakado\temp already exists.
A subdirectory or file C:\Users\Taylor\AppData\Roaming\xirakado\temp\H already exists.
A subdirectory or file C:\Users\Taylor\AppData\Roaming\xirakado\temp\AID already exists.
A subdirectory or file C:\Program already exists.
Error occurred while processing: C:\Program.
A subdirectory or file Files\Xirakado already exists.
Error occurred while processing: Files\Xirakado.
A subdirectory or file rem already exists.
Error occurred while processing: rem.
A subdirectory or file may already exists.
Error occurred while processing: may.
A subdirectory or file be already exists.
Error occurred while processing: be.
A subdirectory or file ommitted already exists.
Error occurred while processing: ommitted.
A subdirectory or file C:\Program already exists.
Error occurred while processing: C:\Program.
A subdirectory or file Files\Xirakado\logs already exists.
Error occurred while processing: Files\Xirakado\logs.
The system cannot find the file specified.
The system cannot find the file specified.

Command Line Error:
Too short switch:
-o

Command Line Error:
Too short switch:
-o
The system cannot find the path specified.
Please check your PSVita, and ensure that your DNS is set to 212.47.229.76 to block updates....then reboot your PSVita
when this is done, launch Content Manager on your PS Vita and connect it to your computer...
Press any key to continue . . .`

**Comment by sephcasiah** [comment]
Okay, I see what I did wrong. CMD commands have an error that can pop up if you try to make folders that exist in a folder the command line isn't run in. Please check your %APPDATA% directory and make sure the xirakado folders have been created, if so, delete them.

**Comment by DanHolli123** [comment]
@sephcasiah I don't mind seeing them now, but is it supposed to create folders named 

> rem

> may

> be

and

> ommited

and what about the "too shor switch: -o" error?

**Comment by sephcasiah** [comment]
no, I'm not sure why those are there. and the "-o" is a command line switch for 7zip, my thinking is it didn't actually send it to 7zip.

**Comment by sephcasiah** [comment]
OH, nevermind, "rem" is a remarks command for cmd that is supposed to comment out lines of text. I see that it did not, in fact, do that.
